catkin_ws/src/dry-test/src/thrusters.py

```diff
@@ -3,10 +3,11 @@
 import rospy
 from auv_msgs.msg import ThrusterCommand
 
-pins = {1:4, 2:5, 3:6, 4:7, 5:8, 6:9, 7:10, 8:11}
-reset = [0, 0, 0, 0, 0, 0, 0, 0]
+pins = {1:2, 2:3, 3:4, 4:5, 5:6, 6:7, 7:8, 8:9}
+reset = [0,0,0,0,0,0,0,0]
 reset_cmd = ThrusterCommand(reset)
-pub = rospy.Publisher('propulsion/thruster_cmd', ThrusterCommand, queue_size=50)
+pub = rospy.Publisher('propulsion/thruster_cmd', ThrusterCommand, queue_size=10)
+rospy.sleep(7)
```

Comment:
why sleep here?

Reply:
To initialize the ESCs. When we create a new publisher, it initializes the topic with all zero values which for some reason re-initializes the ESCs. I put it in to avoid that for now. It could be that the current is too low, which I'll check tomorrow.